hermes/integration/app_test.go

```diff
@@ -0,0 +1,652 @@
+package integration_test
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/cosmos/cosmos-sdk/crypto/keyring"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/ignite/cli/v28/ignite/config/chain"
+	"github.com/ignite/cli/v28/ignite/config/chain/base"
+	v1 "github.com/ignite/cli/v28/ignite/config/chain/v1"
+	pluginsconfig "github.com/ignite/cli/v28/ignite/config/plugins"
+	"github.com/ignite/cli/v28/ignite/pkg/availableport"
+	"github.com/ignite/cli/v28/ignite/pkg/cmdrunner"
+	"github.com/ignite/cli/v28/ignite/pkg/cmdrunner/step"
+	"github.com/ignite/cli/v28/ignite/pkg/goanalysis"
+	yamlmap "github.com/ignite/cli/v28/ignite/pkg/yaml"
```

Comment:
Just made PR ignite/cli#4008 which will affect this import if the other PR is merged

```suggestion
	xyaml "github.com/ignite/cli/v28/ignite/pkg/xyaml"
```

Reply:
We shouldn't update this just yet here. The API break will be released in v29.